calvinextras/calvinsys/web/mqtt/Subscribe.py

```diff
@@ -146,22 +146,40 @@ class Subscribe(base_calvinsys_object.BaseCalvinsysObject):
     }
 
     can_write_schema = {
-        "description": "Does nothing, always return true",
+        "description": "Always return true, allowing configuration of MQTT client",
         "type": "boolean"
     }
 
     write_schema = {
-        "description": "Does nothing"
+        "description": "Update topic subscriptions",
+        "type": "object",
+            "properties": {
+                "topic": {
+                    "type": "string"
+                },
+                "qos": {
+                    "type": "integer"
+                },
+                "cmd": {
+                    "type": "string"
+                },
+            },
+            "required": ["topic"]
```

Comment:
Command would be better as an enum. Also, describe defaults (when  qos and cmd empty.)

Reply:
Done